sdk/ai/ai-projects/test/public/utils/createClient.ts

```diff
@@ -118,15 +118,16 @@ export async function createRecorder(context: VitestTestContext): Promise<Record
 
 export async function createOpenAI(): Promise<OpenAI> {
   const credential = createTestCredential();
-  const projectEndpoint = process.env["OPENAI_PROJECT_ENDPOINT"] || "";
+  const projectEndpoint = process.env["AZURE_AI_PROJECT_ENDPOINT_STRING"] || "";
+  const openAiBaseUrl = `${projectEndpoint}/openai`;
 
   const scope = "https://ai.azure.com/.default";
   const azureADTokenProvider = await getBearerTokenProvider(credential, scope);
 
   return new OpenAI({
     apiKey: azureADTokenProvider,
-    baseURL: projectEndpoint,
-    defaultQuery: { "api-version": "2025-05-15-preview" },
+    baseURL: openAiBaseUrl,
+    defaultQuery: { "api-version": "2025-11-15-preview" },
     defaultHeaders: { "accept-encoding": "deflate" },
```

Comment:
The line defaultHeaders: { "accept-encoding": "deflate" } is no longer needed. It was added to workaround a service bug that was already fixed.